### ISSUE TITLE
Theorie Toets reanimeren burgers

### ISSUE THREAD

**Issue by timfloors** [Tedeapolis/development#360]
**Beschrijf zo duidelijk mogelijk de feature**
Het toevoegen van een theorie toets voor de burgers bij de ambulance om toegang te krijgen tot reanimeren, ook dat dit vervalt na 2 maanden en dat iemand de toets opnieuw moet halen om toegang te krijgen tot reanimeren(plugin van cartheorie gebruiken)

**Wat lost deze feature op?**
Hier voorkom je mee dat mensen alleen maar reanimeren om te reanimeren zonder de roleplay en zonder te weten wat ze eigenlijk doen

**Comment by Willem-Dekker** [comment]
hoe wil je dit gaan doen met de bestaande passen, hebben die geen plicht op nascholing?

**Comment by timfloors** [comment]
dan zal alles gereset worden moeten worden en opnieuw gedaan worden zodat het in een nieuwe systeem kan werken, dit is een idee vanuit de ambu

**Comment by Willem-Dekker** [comment]
ja, maar dit was toen niet duidelijk bij het afnemen van deze examens. dan zou er een zelfde soort regeling moeten komen als bij vliegbrevetten. dat de oude oud blijven en de nieuwe dit systeem gaan gebruiken

**Comment by Willem-Dekker** [comment]
Is dit nog gewenst/haalbaar en hoe staat de ambulancedienst hier in?
